lib/utils.pm

```diff
@@ -682,7 +682,8 @@ sub is_boot_encrypted {
     # encrypted partitions
     # ppc64le on pre-storage-ng boot was part of encrypted LVM
     return 0 if !get_var('FULL_LVM_ENCRYPT') && !is_storage_ng && !is_ppc64le()
-      && !(get_var('ENCRYPT_CANCEL_EXISTING') && get_var('ENCRYPT') && is_aarch64());
+      && !is_sle('15+') && !(get_var('ENCRYPT_CANCEL_EXISTING') && get_var('ENCRYPT')
+        && is_aarch64());
```

Comment:
hm, are you sure that our products changed behaviour at this time? I highly doubt this is true and if it is then it would be a major regression. I mean, for a new major codestream perhaps, but not just during SLE15SP2 development in the middle of two betas

Reply:
Well yes, SLE12 and SLE15 has difference in behavior, even comments above and bellow explain that. The problem is that this condition didn't reflect SLE version at all. Also there was no (as far as i found, you can correct me) test(booting prepared image with lvm+encryption) which would use this combination. This test was went live during SLE12-SP5 and never worked on SLE15-SP2 (which is first SLE15 SP where is is tested).

Reply:
I certainly know how SLE12 and SLE15 differ. Part of the code above is myself. About the latter part I am sure you are right.

@rwx788 you were the last person that changed this line in e802a4dd2 for aarch64. Can you please comment on this?

Reply:
It was 6 months ago, so I don't really remember. This conditions are not maintainable. For old storage stack behavior was totally inconsistent, but for SLE 15 it is.
So if it works, fine. Otherwise I would set some strict expectations for the scenarios where it doesn't work.

Reply:
Also, keep in mind that your change seems to be specific to booting into the image, but it might break fresh installations, so I would do some VRs for it.

Reply:
Installation tests are using different  logic, there is own module `boot_encrypt`. Verification runs were added to description.

Reply:
Alright, with @czerw we figured out what went wrong. In general `!is_sle('15+')` should be a synonym of `!is_storage_ng`, but what we have now is "smart" detection using needle tag to set the variable. Which we obviously don't do in the child test suite. Therefore it didn't work.
So this change is safe, but I would rather put it in `is_storageng` method to make it:
```perl
sub is_storage_ng {
    return get_var('STORAGE_NG') || is_sle('15+');
}
```

Reply:
@rwx788 Thank for discussion, i modified function is_storage_ng. It works as expected.